modules/logging-loki-cli-install.adoc

```diff
@@ -18,7 +18,39 @@ To install and configure logging on your {product-title} cluster, additional Ope
 
 .Procedure
 
-. Create a `Subscription` object:
+--
+include::snippets/logging-stable-updates-snip.adoc[leveloffset=+1]
+--
+
+. Apply the `openshift-operators-redhat` namespace object YAML configuration that follows:
++
+[source,terminal]
+----
+$ oc apply -f openshift_operators_redhat_namespace.yaml
+----
++
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-operators-redhat <1>
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
```

Comment:
Is labels indented properly here? YAMLLINT is unable to confirm this as a proper YAML.